### DISCUSSION TITLE
n/a

### DISCUSSION THREAD

**Discussion by jak5080** [iterate-ch/cyberduck#15656] [Q&A] [answered]

**Comment by dkocher** [comment]
Can you please attach a [debug log](https://docs.cyberduck.io/cyberduck/support/#logging-output) that is recorded from unlocking the vault.

**Comment by jak5080** [comment]


**Reply by dkocher** [reply]
- Have you redacted the debug log?
- Let me know if you find files in your bucket `28d8ab60-ce9f-41b8-8683-9259fec9375d` under `Justin's Backups/d/JD/MWSFJ5FVC57FMIESJQ46TIGFIFSCDX/`.

**Comment by jak5080** [comment]


**Reply by dkocher** [reply]
> Can you please attach a [debug log](https://docs.cyberduck.io/cyberduck/support/#logging-output) that is recorded from unlocking the vault.

Please send the logs to support@cyberduck.io if you are not comfortable attaching them here.

**Comment by ylangisc** [comment, marked as answer]
Looking at your log file the entry point for your vault is `/28d8ab60-ce9f-41b8-8683-9259fec9375d/Justin's Backups/d/JD/MWSFJ5FVC57FMIESJQ46TIGFIFSCDX` which seems to be empty. This folder should contain all root level files and folders. Not sure what happened that it is empty.

You can try to download the whole vault to your local computer and try to open it with the Cryptomator desktop app. This though would help only in case there is a bug in Cyberduck deriving the entry point.
